docs/client-certificate-setup.md

```diff
@@ -0,0 +1,67 @@
+## Client-Certificate based authentication
+
+If the certificate Authority and user certificates are already present the steps of creating the certificate authority and client certificates can be skipped.
+The following is an example of creating them.
+
+```bash
+cd /etc/ssl
+```
+### Create the Certificate Autority (CA)
+
+Firstly, generate the CA:      
+```openssl genrsa -des3 -out ca.key 4096```          
```

Comment:
Even when it is an example, we should only use ciphers that are recommened for new systems in  BSI's TR-02102-1 (https://www.bsi.bund.de/SharedDocs/Downloads/EN/BSI/Publications/TechGuidelines/TG02102/BSI-TR-02102-1.html)    -des3 is not , -aes256 is

Reply:
(Infos now in other document)